src/ResourceManagement/RecoveryServices.Backup/Microsoft.Azure.Management.RecoveryServices.Backup/Customizations/Client.Customizations.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Azure.Management.RecoveryServices.Backup.Models;
```

Comment:
project.json
'- This version should be a preview.  Please append -prview to the version in Newtonsoft.Json
- Consider wither this should be a new library with new namespaces and a new name,m or if it shoudl use the same name as the previous library.


generate.cmd
- broken link to your swagger spec.  Your link should us e a particular commit, and not a link to the branch, like this:
https://raw.githubusercontent.com/Azure/azure-rest-api-specs/188c597d6e87fede4a3b0048ddff3fbfdb7cfd4c/arm-recoveryservicesbackup/compositeRecoveryServicesBackupClient.json

Tests - Your tests are dependnt on pre-created artifacts.  You should have all necessary setup in the test setup.  If this is not possible for some reason, then you need to describe in detail how to create the Azure structures necessary to run the test
SceanrioTests.cs
-= it looks like you are setting the test fixture in each test run.  Please follow the pattern - per-test setup can happen in the test class constructor, per suite setup should happen in a test fixture.